api/tests/unit/routers/test_oauth_refresh.py

```
@@ -88,6 +88,119 @@ async def test_refresh_uses_refresh_token_for_authorization_code_flow(self):
+            patch("src.routers.oauth_connections.resolve_url_template", return_value=provider.token_url),
+            patch("src.services.oauth_provider.OAuthProviderClient.refresh_access_token", new=AsyncMock(return_value=(
+                True,
+                {
+                    "access_token": "refreshed-access-token",
+                    "refresh_token": "new-refresh-token",
+                    "expires_at": datetime.now(timezone.utc) + timedelta(hours=1),
+                },
+            ))) as mock_refresh,
+            patch("src.core.security.decrypt_secret", side_effect=["decrypted-refresh-token", "decrypted-secret"]),
+            patch("src.core.security.encrypt_secret", side_effect=lambda value: f"encrypted:{value}"),
+            patch("src.routers.oauth_connections.CACHE_INVALIDATION_AVAILABLE", False),
```

Ruff (0.15.11)

[warning] Boolean positional value in function call

(FBT003)